### PR TITLE
Added targetgroup and clusterID for ingress impl

### PR DESCRIPTION
**Pull request by Raffo**

### CODE REVIEW

**Comment by Raffo** [comment]
Target group resource must be removed as this is useless.

File: cluster/senza-definition.yaml [hunk 1]
```diff
@@ -250,6 +250,38 @@ Resources:
...
+                  -
+                    Ref: WorkerIAMRole
+        Version: '2012-10-17'
+      Path: /
+      Policies:
+      - PolicyDocument:
+          Statement:
+          - {Action: 'autoscaling:*', Effect: Allow, Resource: '*'}
+          - {Action: 'elasticloadbalancing:*', Effect: Allow, Resource: '*'}
+          - {Action: 'ec2:*', Effect: Allow, Resource: '*'}
```
Comment (hjacobs) [inline cluster/senza-definition.yaml]: do we need to write to ec2? I would limit to needed actions

File: cluster/senza-definition.yaml [hunk 2]
```diff
@@ -250,6 +250,38 @@ Resources:
...
+                -
+                  - arn:aws:iam::{{ AccountInfo.AccountID }}:role/
+                  -
+                    Ref: WorkerIAMRole
+        Version: '2012-10-17'
+      Path: /
+      Policies:
+      - PolicyDocument:
+          Statement:
+          - {Action: 'autoscaling:*', Effect: Allow, Resource: '*'}
```
Comment (Raffo) [inline cluster/senza-definition.yaml]: No, I don't think we need to write there.

File: cluster/senza-definition.yaml (hunk 1 above)
Comment (Raffo) [inline cluster/senza-definition.yaml]: No.

**Comment by aryszka** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**